src/api/annotation/AnnotationAPI.js

```diff
@@ -32,13 +32,15 @@ import _ from 'lodash';
  * @property {String} PIXEL_SPATIAL The pixel-spatial annotation type
  * @property {String} TEMPORAL The temporal annotation type
  * @property {String} PLOT_SPATIAL The plot-spatial annotation type
+ * @property {String} MAP The map annotation type
  */
 const ANNOTATION_TYPES = Object.freeze({
   NOTEBOOK: 'NOTEBOOK',
   GEOSPATIAL: 'GEOSPATIAL',
   PIXEL_SPATIAL: 'PIXEL_SPATIAL',
   TEMPORAL: 'TEMPORAL',
-  PLOT_SPATIAL: 'PLOT_SPATIAL'
+  PLOT_SPATIAL: 'PLOT_SPATIAL',
+  MAP: 'MAP'
```

Comment:
I think the point of `GEOSPATIAL` was the same as `MAP`, so we should pick one or the other.

Reply:
Ah, good to know! I initially had those marked as `GEOSPATIAL` type, but I wasn't sure if we were planning on using these for anything else, so I added in the `MAP` type along with the special handling in `AnnotationSearchResult`. I'll switch it back.